src/main/scala/org/broadinstitute/dsde/rawls/webservice/UserApiService.scala

```diff
@@ -24,6 +22,15 @@ trait UserApiService extends HttpService with PerRequestCreator with UserInfoDir
   val userServiceConstructor: UserInfo => UserService
 
   val userRoutes = requireUserInfo() { userInfo =>
+    path("user") {
+      post {
+        entity(as[RawlsUser]) { userToCreate =>
```

Comment:
I think I was confused about whether this was an admin endpoint, and therefore required a different user than the OIDC headers.

But no, that is not the case, so OIDC is fine.  I'll update it to work that way.